docs/configuration.rst

```diff
@@ -697,10 +696,8 @@ PAPERLESS_IGNORE_DATES=<string>
     this process. This is useful for special dates (like date of birth) that appear
     in documents regularly but are very unlikely to be the documents creation date.
 
-    You may specify dates in a multitude of formats supported by dateparser (see
-    https://dateparser.readthedocs.io/en/latest/#popular-formats) but as the dates
-    need to be comma separated, the options are limited.
-    Example: "2020-12-02,22.04.1999"
+    You must specify dates in the same date ordering as PAPERLESS_DATE_ORDER is
+    set to or its default ordering.
```

Comment:
I'm a little dense so please bear with me -- This means if `PAPERLESS_DATE_ORDER` is set to `DMY`, `PAPERLESS_IGNORE_DATES` must also follow this convention? So this would be valid:

```env
PAPERLESS_DATE_ORDER=DMY
PAPERLESS_IGNORE_DATES=26/11/2021
```

Wheras this would not be valid:

```env
PAPERLESS_DATE_ORDER=MDY
PAPERLESS_IGNORE_DATES=26/11/2021
```

Does anything break in the invalid case? Or the ignore just isn't honored?

Reply:
Hah okay that's good to know! So then can we make this particular line more accurate by saying something like:

The date is parsed using the format specified in `PAPERLESS_DATE_ORDER`.

Reply:
Updated